raptor/playback.py

```diff
@@ -0,0 +1,38 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from __future__ import absolute_import
+
+import os
+
+from raptor.playback_tools.mitmproxy.mitmproxy import Mitmproxy
```

Comment:
I think these packages might be unnecessarily nested. Maybe try something like:

- raptor
    - tools
        - \_\_init\_\_.py
        - mitmproxy.py

Then in \_\_init\_\_.py, you can do `from mitmproxy import Mitmproxy` which leads to a final import of `from raptor.tools import Mitmproxy`. Not sure what `playback` means, but I guess if there will be lots of `playback` related tools, you could keep the name `playback_tools`. It's bit verbose sounding though.